.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

Comment:
You can probably get rid of `master` and just have `branches: main` (I forget if you need the brackets if you just have 1 branch)